Add radmonflag

cegrant opened the pull request:
Added a draft anomaly page for the radmon flag.  Ready for reviews and edits from the team.

Commit:
New radmon flag page

cegrant committed:
Update radmon_flag.rst

A few more edits

richardjedgar commented on source/radmon_flag.rst:
```diff
@@ -0,0 +1,120 @@
+
+* While the Radiation Monitor Flag is asserted, ACIS will not take science data.
+* When the Radiation Monitor Flag is de-asserted, ACIS will re-start the suspended science run, or start a new science run, depending on the ACIS commands that may have been received while the monitor was asserted.
+
```
So if several observations have been missed, what does ACIS do when the flag is de-asserted? Presumably it has finite memory for commands not executed.

cegrant replied:
From our discussion yesterday, I believe ACIS will start up the most recent configuration it was commanded to start.  (Or will start nothing if the last commanding was a stop science).  Peter & Jim need to confirm much of this on the EU.  (And should review this page once it gets more filled out.)

jazan12 commented on source/radmon_flag.rst:
```diff
@@ -0,0 +1,120 @@
+
+* Send an email to the ACIS team at the official anomaly email address, ``acis-anomaly -at- googlegroups -dot- com``.
+  If it is off-hours, another ACIS Ops team member should call Peter
+  Ford, Jim Francis, and Bob Goeke.
```
If we think this is the event going on, does Bob really need to be part of the initial software checks?

cegrant replied:
I copied the notifications from another anomaly page and we could certainly discuss who gets notified when.  If we're diagnosing an unknown or not seen before anomaly, I think we would want the whole brain trust including Bob.  He may not be as expert as Peter and Jim, but is still more expert than the rest of us.

jazan12 commented on source/radmon_flag.rst:
```diff
@@ -0,0 +1,120 @@
+  radiation monitor flag has been enabled and that science data will
+  not be collected until it is disabled.
+* Recommend to the Lead Systems Engineer and the Flight Director to
+  send the command 1RMONIRM (v=0) to
```
Equivalent bcmd "set radiationmonitor low".  (Which do they use in practice?)

cegrant replied:
It's a hardware command, so the FOT needs the actually command.  https://acis.mit.edu/acis/ipcl/ipcl-cmds-08jun98.html  I'm not sure what the equivalent bcmd is for. Can you tell me where you saw it?

jazan12 commented on source/radmon_flag.rst:
```diff
@@ -0,0 +1,120 @@
+ACIS Commands
++++++++++++++
+
+* ``1RMONIRM(0)`` De-assert Radiation Monitor Flag
```
same note as above regarding "set radiationmonitor low"

cegrant replied:
same as above  ;-)

jazan12 commented on source/radmon_flag.rst:
```diff
@@ -0,0 +1,120 @@
+
+Spurious high level pulse commands to the PSMC have caused anomalies, but there have been no spurious serial digital commands as of yet.
+
+Note that the radiation monitor flag discussed here is entirely different from the TXINGS patch, which uses ACIS data to detect high radiation environment and then sets the ACIS bilevels to a particular pattern for the OBC to act upon.  The ACIS radiation monitor flag is not used in current operation at all.
```
Suggest moving this up slightly to under "Relevant Notes/Memos".

cegrant replied:
I actually thought about putting it all the way at the top of the page, since the potential confusion starts with the name of the anomaly.  What do you think?

jazan12 reviewed (changes requested):
Looks good!  I had a few small things I commented on in there.

jazan12 commented:
Hey Catherine,

Thanks for the replies.  I nabbed that bcmd from the FSWUG, Section 3.9,
this table: Radiation Alert Hardware Serial Digital Command Mnemonics.  I
think your suggestion to move that warning note up to the top of the page
sounds just right.

-Jack





On Mon, Dec 14, 2020 at 7:35 PM cegrant <notifications@github.com> wrote:
>
> @cegrant commented on this pull request.
>
> ________________________________
>
> In source/radmon_flag.rst:
>
> > +
> +What is the response?
> +---------------------
> +
> +(will this show up in a standard SOH by OC/CC?  Don't think so but
> +need to check)
> +We need to:
> +
> +* Send an email to the ACIS team at the official anomaly email address,
``acis-anomaly -at- googlegroups -dot- com``.
> +  If it is off-hours, another ACIS Ops team member should call Peter
> +  Ford, Jim Francis, and Bob Goeke.
> +* Send an email to ``sot_red_alert@cfa`` announcing that the ACIS
> +  radiation monitor flag has been enabled and that science data will
> +  not be collected until it is disabled.
> +* Recommend to the Lead Systems Engineer and the Flight Director to
> +  send the command 1RMONIRM (v=0) to
>
> It's a hardware command, so the FOT needs the actually command.
https://acis.mit.edu/acis/ipcl/ipcl-cmds-08jun98.html I'm not sure what the
equivalent bcmd is for. Can you tell me where you saw it?
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub, or unsubscribe.

>

cegrant committed:
Update index.rst

cegrant committed:
Update radmon_flag.rst

Further updates including
- moving the TXINGS note to the top of the page
- adding the expected DEA current, DPA current, and 1STAT1ST
- adding text on the FOT state of health check (which I do not think would see this anomaly)

cegrant commented:
Thanks Jack, I've moved the TXINGS note to the top and made a few more text changes.
